### DISCUSSION TITLE
120Hz support for Quest 2

### DISCUSSION THREAD

**Discussion by mavanmanen** [alvr-org/ALVR#608] [General]
Is this feature on the way?

**Comment by sctanf** [comment]
You can change preferredFPS to 120 in session.json.
However there is currently a bug limiting the client FPS to 105-107.

**Reply by mavanmanen** [reply]
Ahh, too bad about the bug.

**Reply by sctanf** [reply]
I heard some fixes were pushed, ALVR nightly should be able to use 120hz.
